src/features/click-to-play.js

```diff
@@ -20,6 +20,9 @@ let sharedStrings = null
 const entities = []
 const entityData = {}
 
+let readyResolver
+const ready = new Promise(resolve => { readyResolver = resolve })
```

Comment:
(This is used so that we can block replacing placeholders only until the feature has finished starting up.)